### DISCUSSION TITLE
STUDENT PACK AUTH

### DISCUSSION THREAD

**Discussion by Simpenzwe-Honore-Leandre** [community/community#31852] [GitHub Education] [answered]
@github-staff  Hello, I am currently trying to authenticate my status as a student. My ID card has two sides and the date indicating enrollment is on the back. If I upload that side the other one won't be visible, any suggestions?

**Comment by melvincwng** [comment]
@LeandreH, are you able to scan both sides of your student ID and combine them into one JPG/PNG file. This can be done easily using photo editor software like Paint or Photoshop.

Then try uploading that combined ID JPG/PNG file instead.

Hope this works and answers your question 😊

**Reply by Simpenzwe-Honore-Leandre** [reply]
Thanks for the prompt reply😊, I can scan them, but there seems to be no button for uploading files directly from the device. If you could guide me on how it can be done, I would appreciate it.

**Comment by melvincwng** [comment, marked as answer]
@LeandreH, seems like the 'Upload image' function has been removed in GitHub Education.

In that case, I would suggest you try one of the following two options below:

1. Submit your application using one of the other documents seen below (choose one besides the student ID):
![image](https://user-images.githubusercontent.com/77479885/188276652-ceebcfde-4444-403e-8403-84a6ad4d8622.png)

3. If option 1 is not available to you, try submitting your application for the GitHub Student Developer Pack using your student ID (you can choose either the front or back, and see if GitHub approves your application).

If all fails, contact [GitHub Education Support](https://support.github.com/contact/education) over here and explain the issue you are facing. You can link this discussion thread to the support engineers at GitHub Education Support, so as to provide more context to them on this issue you are facing.

Hope this helps you in your application.

**Reply by melvincwng** [reply]
@LeandreH , just thought of a third option/suggestion:

3. If you are going ahead with option 2 (application using student ID), you can also try this:
- Try scanning a combined picture of the front & back of your student ID & then print it out
- Hold your student ID (choose either front or back) & hold the printed combined picture
- Take the photo (showing your face, actual student ID, and printed version of front + back of student ID).

Hopefully this approach provides sufficient evidence to GitHub that you are a valid student at your institution, and thus speeding up the approval process for the GitHub Student Developer pack.

Take care, and all the best in your application 🙏🏻

**Reply by Simpenzwe-Honore-Leandre** [reply]
@melvincwng thank you🙏! I have sent a ticket to GitHub support, so I'll wait to see what they have to say before moving on to the other options.

**Reply by melvincwng** [reply]
@LeandreH, no problem!

Good luck with your application, and have a nice day ahead 😊
